ofnetAgent.go

```diff
@@ -569,7 +569,18 @@ func (self *OfnetAgent) RemoveNetwork(vlanId uint16, vni uint32, Gw string, Vrf
 	// make sure there are no endpoints still installed in this vlan
 	for _, endpoint := range self.endpointDb {
```

Comment:
I think this block needs to be moved to the beginning of this function. i.e. clean up the endpoints before performing any other delete action.